Formula/httpd.rb

```diff
@@ -140,17 +141,19 @@ def plist; <<-EOS.undent
         LoadModule authz_core_module #{lib}/httpd/modules/mod_authz_core.so
         LoadModule unixd_module #{lib}/httpd/modules/mod_unixd.so
         LoadModule dir_module #{lib}/httpd/modules/mod_dir.so
-        LoadModule mpm_event_module #{lib}/httpd/modules/mod_mpm_event.so
+        LoadModule mpm_prefork_module #{lib}/httpd/modules/mod_mpm_prefork.so
       EOS
 
       pid = fork do
-        exec bin/"httpd", "-DFOREGROUND", "-f", "#{testpath}/httpd.conf"
+        exec bin/"httpd", "-DNO_FORK", "-f", testpath/"httpd.conf"
       end
-      sleep 3
+      sleep 8
+      # calculate parent pid
+      parent_pid = pid + 1
```

Comment:
This isn't a safe assumption. The PID should/could be written out to a file which feels like a safer way of doing this if it's impossible to do so with `-DFOREGROUND`

Reply:
The other option here is to use `-X`

```
-X                 : debug mode (only one worker, do not detach)
```

Reply:
I meant to note that I didn't think this to be a real solution, but rather it pointed out what some of the current issues were.